OutSystems/outsystems-datagrid-reactive
# ROU-3981: Fix issue when setting Min and Max limits to zero

This PR is to fix an issue when setting the minimum and maximum values on the column and these values are zero.

### What was happening
* When setting the minimum and maximum values on the column and this value is zero the column was allowing negative numbers.

### What was done
* Added extra validations to cover the scenario when the minimum and maximum values on the column values are zero.

### Test Steps
1. Test Case 1 - Regression ROU-3213:
     - Set Decimal places = 6
     - Selected a random cell from column Price
     - Typed F2
     - Typed 9999999999
     - Pressed Enter
     - Got the Expected results with:
         - Cell value = 8,589,934,591
         - Column Price present currency symbol"

2. Test Case 2 - Reported scenario:
     - Created a screen with Number and Currency columns having MinVlaue/MaxValue/DecimalPlaces/CurrencySymbol being set both directly on the column properties and as variables that we can change in runtime
     - On each cell, try to input a number out of bounds.
     - Checked that is not allowed and that it will be reset to the min or max value accordingly
     - Manipulate Min/Max, Currency symbol and decimal places in runtime
     - On each cell, try to input a number out of bounds.
     - Checked that is not allowed and that it will be reset to the min or max value accordingly


### Checklist
* [ ] requires a new sample page in OutSystems
